db/migrate/20170717154257_add_ticket_confirmation_to_email_settings.rb

```diff
@@ -0,0 +1,7 @@
+class AddTicketConfirmationToEmailSettings < ActiveRecord::Migration
+  def change
+    add_column :email_settings, :send_on_ticket_confirmation, :boolean, default: false
```

Comment:
>  And I think we don't need to customize the message.

So should set the message manually or like use the same message that sent in registration confirmation email?